poshc2/client/command_handlers/FCommHandler.py

```diff
@@ -8,7 +8,7 @@
 from poshc2.Colours import Colours
 from poshc2.Utils import validate_sleep_time, argp, load_file, gen_key
 from poshc2.server.AutoLoads import check_module_loaded, run_autoloads_sharp
-from poshc2.client.Help import sharp_help
+from poshc2.client.Help import py_help, sharp_help, posh_help, server_help
```

Comment:
Better to add them together in `Help.py` and expose a variable called `all_help` or similar, which we then import in the relevant files.

That way if we need to add/remove things we do it all in Help.py and the changes will automatically propagate to the other files.

Reply:
Yeah good point - I've merged them in help.py and added that in each handler.